### DISCUSSION TITLE
계단산

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#7] [Algorithm]
- 계단으로 된 산인 계단산 입구에는 등고선과 함께 다음과 같은 안내문이 적혀있다.
![계단산](https://user-images.githubusercontent.com/59595261/130245296-eae54a52-92aa-4e90-836f-ef7325b1927e.png)
- 위와 같은 등고선이 주어졌을 때 그려지는 계단산은 아래와 같으며 소모되는 칼로리는 48kcal이다.
![계단산2](https://user-images.githubusercontent.com/59595261/130245380-a374751b-c4e1-4c54-b888-932371ff40fb.png)
- 등고선이 주어질 때 얼마의 칼로리를 소모하는지 계산하라.

<br>

### 입력 설명
첫째 줄에는 계단산의 등고선이 주어진다.
### 입력 예시
`((#(())#(())))`
### 출력 예시
`48`

**Comment by pyunji** [comment]
[solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/pyunji/homework/210820_problem2.java)

**Reply by hyk339** [reply]
beautiful code!!

**Comment by hyk339** [comment]
[solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/homework/210820_Problem2.java)

**Comment by chaeyeon4u** [comment]
[solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/chaeyeon/homework/Java/0820_Prob2.java)

**Comment by leechang4** [comment]
[solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/0821_Homework2.JAVA)
